scanners/subfinder/examples/securecodebox.io/active/scan.yaml

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: "execution.securecodebox.io/v1"
```

Comment:
nice examples 🙌
might be good to add an explanation how the active and passive scans differ to make this clearer to people looking at the examples